test/parser/pg_parser_test.cpp

```diff
@@ -0,0 +1,77 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// parser_test.cpp
+//
+// Identification: test/parser/parser_test.cpp
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include <string>
+#include <vector>
+
+#include "common/harness.h"
+#include "common/macros.h"
+#include "common/logger.h"
+#include "parser/pg_parser.h"
+#include "parser/parser.h"
+
+namespace peloton {
+namespace test {
+
+//===--------------------------------------------------------------------===//
+// Parser Tests
+//===--------------------------------------------------------------------===//
+
+class PgParserTests : public PelotonTest {};
```

Comment:
Actually, what if you just renamed it to `PostgresParserTests`? Use `Postgres` instead of `Pg` so that it is more clear.